hogben/simulate.py

```diff
@@ -1,101 +1,157 @@
+import os.path
+from typing import Optional, Union
+
 import importlib_resources
+import numpy as np
 
 import refnx.reflect
 import refl1d.model
 import refl1d.probe
 import refl1d.experiment
 
-import numpy as np
 
+def direct_beam_path(inst_or_path: str = 'OFFSPEC',
+                     polarised: bool = False) -> Union[str, None]:
```

Comment:
The function doesn't return `None` any more, maybe `Union[str, Path]`? (not sure what type `importlib_resources.files('hogben.data.directbeams').joinpath` returns). Last comment I promise...

Reply:
I'm pretty sure it is a string. And thanks for spotting this!